config/application.rb

```diff
@@ -16,6 +16,7 @@ class Application < Rails::Application
     config.i18n.available_locales = [:en, :vi]
     config.i18n.default_locale = :en
 
+    config.email_regex = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
```

Comment:
cái này là config cho toàn ứng dụng nè e, mình chỉ cần define constant trong file settings.yml là đc nha